code/data_structures/src/stack/infix_to_prefix/Infix_To_Prefix.cpp

```diff
@@ -0,0 +1,139 @@
+{
+    // stack for operators.
+    stack<char> operators;
+    // stack for operands.
+    stack<string> operands;
+
+    for (int i = 0; i < infix.length(); i++)
+    {
+        /** If current character is an
+        opening bracket, then
+        push into the operators stack. **/
+        if (infix[i] == '(')
+            operators.push(infix[i]);
+
+
+        /** If current character is a
+            closing bracket, then pop from
+            both stacks and push result
+            in operands stack until
+            matching opening bracket is
+            not found. **/
+        else if (infix[i] == ')')
+        {
+            while (!operators.empty() && operators.top() != '(')
+            {
+                // operand 1
+                string op1 = operands.top();
+                operands.pop();
+                // operand 2
+                string op2 = operands.top();
+                operands.pop();
+                // operator
+                char op = operators.top();
+                operators.pop();
+                /** Add operands and operator
+                   in form operator +
+                   operand1 + operand2. **/
+                string tmp = op + op2 + op1;
+                operands.push(tmp);
+            }
+
+            /** Pop opening bracket from stack. **/
+            operators.pop();
+        }
+
+        /** If current character is an
+            operand then push it into
+            operands stack. **/
+        else if (!isOperator(infix[i]))
+            operands.push(string(1, infix[i]));
+
+
+        /** If current character is an
+            operator, then push it into
+            operators stack after popping
+            high priority operators from
+            operators stack and pushing
+            result in operands stack. **/
+        else
+        {
+            while (!operators.empty() && getPriority(infix[i]) <= getPriority(operators.top())) {
+                string op1 = operands.top();
+                operands.pop();
+                string op2 = operands.top();
+                operands.pop();
+                char op = operators.top();
+                operators.pop();
+                string tmp = op + op2 + op1;
+                operands.push(tmp);
+            }
+            operators.push(infix[i]);
+        }
+    }
+
+    /** Pop operators from operators stack
+        until it is empty and add result
+        of each pop operation in
+        operands stack. **/
+    while (!operators.empty())
+    {
+        string op1 = operands.top();
+        operands.pop();
+
+        string op2 = operands.top();
+        operands.pop();
+
+        char op = operators.top();
+        operators.pop();
+
+        string tmp = op + op2 + op1;
+        operands.push(tmp);
+    }
+
+    /** Final prefix expression is
+        present in operands stack. **/
+    return operands.top();
+}
+
+// Driver code
+int main()
+{
+    string s;
+    cin >> s;
+    cout << infixToPrefix(s);
```

Comment:
Since `using namespace std` is not being used, change these to `std::cin` and `std::cout`.